### DISCUSSION TITLE
Is it still possible to use v4.x with "plain" PostgreSQL?

### DISCUSSION THREAD

**Discussion by dennisconrad** [ludeknovy/jtl-reporter#242] [Q&A] [answered]
The docker-compose.yml in version 4.x uses PostgreSQL with the Timescale extension.  Is it still possible to use "plain" PostgreSQL with version 4.x?  Thank you!

**Comment by ludeknovy** [comment, marked as answer]
Hi @dennisconrad ! I don't think so. If I remember correctly, there are a few timescale db functions used in the queries.
Eg: https://docs.timescale.com/api/latest/hyperfunctions/histogram/

Is there any issue with the timescale?

**Reply by dennisconrad** [reply]
Thank you for your swift response!  There are no issues with Timescale, we are just running JTL Reporter as a Task in AWS ECS, and currently use an RDS instance w/ PostgreSQL as our DB instance.
